src/lib/include/roc/address.h

```diff
@@ -85,6 +85,22 @@ typedef struct roc_address {
 ROC_API int
 roc_address_init(roc_address* address, roc_family family, const char* ip, int port);
 
+/** Set multicast interface address.
+ *
+ * @b Parameters
+ *  - @p address should point to a properly initialized address struct
+ *  - @p family should be @c ROC_AF_AUTO, @c ROC_AF_IPv4, or @c ROC_AF_IPv6
+ *  - @p iface should point to a zero-terminated string with a valid IPv4 or
+ *    IPv6 multicast address
+ *
+ * @b Returns
```

Comment:
`@b Returns` -> `@returns` ?

Reply:
Look at another `Returns`. I did the same.

Reply:
Hm, maybe I did it because it's a list.